src/components/Button.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+
+const Button = ({ buttonName }) => (
+  <button type="button" value={buttonName}>{buttonName}</button>
+);
+
+export default Button;
```

Comment:
make `buttonName` be `STRING` using propTypes [ [see requirements](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/validateAsString.png) ]